docroot/themes/custom/twentyseventeen/sass/base/elements/_elements.scss

```diff
@@ -55,6 +58,7 @@ a,
 a:link,
 a:visited {
   text-decoration: none;
+  //color: rgba(134,151,58,1);
```

Comment:
Please remove this.  We try to avoid committing commented code because it never gets removed.